src/lua/minifio.c

```diff
@@ -53,6 +53,12 @@ tarantool_lua_minifio_init(struct lua_State *L)
 		{NULL, NULL}
 	};
 
+	/*
+	 * The loaders.builtin loader is not in effect yet.
+	 * Set the minifio module into package.loaded manually.
+	 */
+	lua_getfield(L, LUA_REGISTRYINDEX, "_LOADED");
 	luaT_newmodule(L, "internal.minifio", minifio_methods);
+	lua_setfield(L, -2, "internal.minifio");
 	lua_pop(L, 1);
```

Comment:
`lua_pop` looks excess here (`lua_setfield` removes the module field from the stack).

Reply:
It pops `_LOADED`.

Reply:
PEBKAC. Anyway, comment desired, if you don't mind.

Reply:
No problem, added.

```diff
diff --git a/src/lua/minifio.c b/src/lua/minifio.c
index 952ad86e2..7795408a8 100644
--- a/src/lua/minifio.c
+++ b/src/lua/minifio.c
@@ -60,5 +60,5 @@ tarantool_lua_minifio_init(struct lua_State *L)
        lua_getfield(L, LUA_REGISTRYINDEX, "_LOADED");
        luaT_newmodule(L, "internal.minifio", minifio_methods);
        lua_setfield(L, -2, "internal.minifio");
-       lua_pop(L, 1);
+       lua_pop(L, 1); /* _LOADED */
 }
```

Reply:
Thanks!